@navikt/core/react/src/table/stories/table.stories.tsx

```diff
@@ -20,18 +21,48 @@ export const All = () => {
       </Table.Header>
       <Table.Body>
         <Table.Row>
+          {props.button && (
+            <Table.DataCell
```

Comment:
Har vi en måte forenkle sentrering av noe i tabell? Gjelder her for knapp, men også for eks ikoner.

Reply:
Bør man kunne velge selv om knapper skal være sentrert eller ikke? Vi har et case hvor en kolonne kan inneholde både navn på saksbehandler hvis en sak er under behandling og en tertiær "start saken" knapp. 
Da ønsker vi at all tekst kommer på en vertikal linje uavhengig av om det er teksten i knappen eller saksbehandlers navn

Reply:
Jeg forstår det som at Ken snakker om vertikal sentrering, og det er en vanskelig case når elementer er av ulik høyde og man ønsker en kompakt tabell der hver rad er kun 36px høy. Jeg har ikke klart å finne noen god løsning enda, så jeg er veldig åpen for forslag.

Reply:
Tenkte Horisontal sentrering, eks:  | *     |, |   *   |, |     * |

Reply:
Er ikke det enkelt å gjøre i innholdet med enten `text-align` eller noe grid eller flexbox?

Reply:
Er relativt lett med bruk av className og css. Men tenker kanskje dette er en så vanlig case at det kan bygges inn i table cellene?

Reply:
Jeg vil tro alignment oftest skjer på kolonnenivå. Hva med en løsning som denne? https://react-spectrum.adobe.com/react-spectrum/TableView.html#column-alignment

Reply:
Ja en slik løsning har vært perfekt!